src/printer.ts

```diff
@@ -547,11 +564,8 @@ export class PugPrinter {
 			if (isQuoted(token.val)) {
 				if (token.name === 'class') {
 					// Handle class attribute
-					let val: string = token.val;
-					val = val.slice(1, -1);
-					val = val.trim();
-					val = val.replace(/\s\s+/g, ' ');
-					const classes: string[] = val.split(' ');
```

Comment:
When looking for use-cases `replace()`, I found these lines and realized they could be optimized.